packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/RNReactNativeGutenbergBridgeModule.java

```diff
@@ -162,6 +163,11 @@ public void updateTheme(@Nullable Bundle editorTheme) {
         emitToJS(EVENT_NAME_UPDATE_EDITOR_SETTINGS, writableMap);
     }
 
+    public void showEditorHelp() {
+        WritableMap writableMap = new WritableNativeMap();
```

Comment:
Whenever we finish this work, what will the contents of `writeableMap` be? Do we need to send any information or is it merely that we are queried to include this parameter for the JavaScript event? 

Relatedly, I was looking to accomplishing something similar — [create an empty object for a JavaScript callback](https://github.com/WordPress/gutenberg/pull/33119/files#diff-7eaacb31e4eb9acc12388550403d16bdd0e7bfa60ba442adc0c14ed9abff4c38R247) — and utilized `Arguments` instead. I wonder the differences between these two approaches. How did you land on `WritableNativeMap`? I learned of `Arguments` from [this documentation](https://reactnative.dev/docs/native-modules-android#sending-events-to-javascript).

Reply:
The contents will likely always be empty. I had chosen to do it this way because I saw another method in the same class do the same thing. Looking at `Arguments.createMap()`, [it looks like it does exactly the same thing in the underlying code](https://github.com/facebook/react-native/blob/d5676f83705275b8a109c361e6dde8be5ea5c98b/ReactAndroid/src/main/java/com/facebook/react/bridge/Arguments.java#L158). That said, I didn't realize I could pass a `null` until just now so I've updated it to do that instead in 26fcb14a277975bd11371c29c997b15681957821